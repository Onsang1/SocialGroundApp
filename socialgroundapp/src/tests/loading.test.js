/**
* @jest-environment jsdom
*/

import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import '@testing-library/jest-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { expect, test } from '@jest/globals';
import renderer from 'react-test-renderer';
import Loading from '../components/Loading';

test('renders loading image', () => {
  render(
    <Router>
      <Loading />
    </Router>,
  );
  expect(screen.getByTestId('loading')).toBeInTheDocument();
});

test('test to see if loading components are rendered', async () => {
  const tree = renderer.create(<Loading />);
  expect(tree).toMatchSnapshot();
});
