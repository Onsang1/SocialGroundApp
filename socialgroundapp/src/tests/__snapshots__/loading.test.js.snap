// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if loading components are rendered 1`] = `
<img
  alt="loading"
  data-testid="loading"
  id="loading"
  src={Object {}}
/>
`;
